app/Http/Resources/Editable/EditableBaseAlbumResource.php

```diff
@@ -43,7 +43,7 @@ class EditableBaseAlbumResource extends Data
 	public bool $is_model_album;
 	public bool $is_pinned;
 
```

Comment:
Untagged albums do not have the EditableBaseAlbumResource. That one is used to specify the "description" and other configurations...

And Untagged album is effectively a smart album this one won't be populated.